core/threaded/reactor_threaded.c

```diff
@@ -1137,8 +1137,23 @@ int lf_reactor_c_main(int argc, char* argv[]) {
 
     if (process_args(default_argc, default_argv)
             && process_args(argc, argv)) {
-        lf_mutex_lock(&mutex); // Sets start_time
-        initialize();
+        
+        // If _lf_number_of_threads is 0, it means that it was not provided on
+        // the command-line using the --threads argument.
```

Comment:
Is this comment correct? First, should the command-line argument name be changed to match the target property name? Second, what if this is specified as a target property rather than a command-line argument?

Reply:
It thought that if the target property is specified, this results in the `#define` to be set in the generated code. We have to make sure that the CLI arg still overrides it, but it seems like this would indeed be that case...

Reply:
> First, should the command-line argument name be changed to match the target property name?

Ah, yes, thank you. It should be workers. I'll make that change.

> Second, what if this is specified as a target property rather than a command-line argument

The command-line argument takes priority. This made the most sense to me at least.

Reply:
There is a caveat to that approach: if there is a minimum number of workers required to accommodate blocking input control reactions, we don't want to go below that number. I think we need to encode that requirement separately.

Reply:
Yes, good point. I applied your suggestion.